modules/crud/sql/describe.go

```diff
@@ -2,27 +2,56 @@ package sql
 
 import (
 	"context"
-	"fmt"
 
 	"github.com/spaceuptech/space-cloud/utils"
 )
 
 // DescribeTable return a description of sql table & foreign keys in table
 // NOTE: not to be exposed externally
-func (s *SQL) DescribeTable(ctx context.Context, project, col string) ([]utils.FieldType, []utils.ForeignKeysType, error) {
-	fields, err := s.getDescribeDetails(ctx, col)
+func (s *SQL) DescribeTable(ctx context.Context, project, dbType, col string) ([]utils.FieldType, []utils.ForeignKeysType, error) {
+	fields, err := s.getDescribeDetails(ctx, project, dbType, col)
 	if err != nil {
 		return nil, nil, err
 	}
-	foreignKeys, err := s.getForeignKeyDetails(ctx, project, col)
+	foreignKeys, err := s.getForeignKeyDetails(ctx, project, dbType, col)
 	if err != nil {
 		return nil, nil, err
 	}
 	return fields, foreignKeys, nil
 }
 
-func (s *SQL) getDescribeDetails(ctx context.Context, col string) ([]utils.FieldType, error) {
-	rows, err := s.client.Queryx("DESCRIBE " + col)
+func (s *SQL) getDescribeDetails(ctx context.Context, project, dbType, col string) ([]utils.FieldType, error) {
+	queryString := ""
+	if utils.DBType(dbType) == utils.MySQL {
+		queryString = `DESCRIBE ` + project + "." + col
+	} else {
+		queryString = `SELECT  
```

Comment:
Which database is this searching for the table in?

Reply:
query needs to be changes for that, will solve tmr with ur help